### PR TITLE
Add `GET /api/v1/me/tokens/:id` API endpoint

### PR DESCRIPTION
**Pull request by hi-rustin**
ref https://github.com/rust-lang/crates.io/issues/8717

In this PR I have added the `show` API for ApiToken to prepare to get the information by ID on the create token page.

And I also added some tests.

### CODE REVIEW

File: src/router.rs [hunk 1]
```diff
@@ -104,6 +104,7 @@ pub fn build_axum_router(state: AppState) -> Router<()> {
         .route("/api/v1/me", get(user::me::me))
         .route("/api/v1/me/updates", get(user::me::updates))
         .route("/api/v1/me/tokens", get(token::list).put(token::new))
+        .route("/api/v1/me/tokens/:id", get(token::show))
         .route("/api/v1/me/tokens/:id", delete(token::revoke))
```
Comment (Turbo87) [inline src/router.rs:108]:
```suggestion
        .route("/api/v1/me/tokens/:id", get(token::show).delete(token::revoke))
```

File: src/tests/routes/me/tokens/get.rs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        ".api_token.expired_at" => "[datetime]",
+    });
+}
+
+#[tokio::test(flavor = "multi_thread")]
+async fn show_with_anonymous_user() {
+    let url = "/api/v1/me/tokens/1";
+    let (_, anon) = TestApp::init().empty();
+    anon.get(url).await.assert_forbidden();
+}
```
Comment (Turbo87) [inline src/tests/routes/me/tokens/get.rs:73]: can you add another test where the token belongs to user A and user B is trying to request it? the existing test triggers the `AuthCheck`, but the new one would check the `belongs_to()`.

**Review by Turbo87 — COMMENTED** [review]
looks good! :)

File: src/tests/routes/me/tokens/get.rs (hunk 2 above)
Comment (hi-rustin) [reply src/tests/routes/me/tokens/get.rs:73]: oh, nice! I will add it.

Comment (hi-rustin) [reply src/tests/routes/me/tokens/get.rs:73]: Added.

File: src/router.rs (hunk 1 above)
Comment (hi-rustin) [reply src/router.rs]: Updated.

**Review by hi-rustin — COMMENTED** [review]
🔢 Self-check (PR reviewed by myself and ready for feedback.)

**Review by Turbo87 — APPROVED** [review]

**Pull request merged**